redis/src/sentinel.rs

```diff
@@ -151,7 +163,10 @@ impl SentinelNodeConnectionInfo {
                 host: ip,
                 port,
                 insecure: false,
+                #[cfg(not(feature = "tls-rustls"))]
                 tls_params: None,
+                #[cfg(feature = "tls-rustls")]
+                tls_params: Some(retrieve_tls_certificates(self.certs.clone().unwrap()).unwrap()),
```

Comment:
let's not use `unwrap` here, and instead match on `self.certs.clone()`, and return error on a failed result of `retrieve_tls_certificates`.